recipes/aicspylibczi/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "aicspylibczi" %}
+{% set version = "3.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/aicspylibczi-{{ version }}.tar.gz
+  sha256: 435d1544421cacc05838aa31afaade85da0972bd3638bfe008fae6e023248ad0
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+  host:
+    - python
+    - setuptools >=42
+    - cmake >=3.18
+    - ninja
```

Comment:
Ninja and cmake should be in build

Reply:
Hmm, that part was generated by `grayskull`, I wonder why it did that.